dev/io.openliberty.org.jboss.resteasy.common.ee10/resources/META-INF/services/jakarta.ws.rs.core.EntityPart$Builder

```diff
@@ -0,0 +1,20 @@
+#
```

Comment:
Isn't this already brought in with Include-Resource in the bnd file?

Reply:
Same as previous comment.